README.md

```diff
@@ -8,9 +8,22 @@
 In addition, the proxied web page does not know that it is opened under a proxy. The proxy intercepts access attempts to all URL-containing properties and provides the original values.
 
 ## First Look
-1. Clone the Hammerhead repository (`git clone https://github.com/DevExpress/testcafe-hammerhead.git`)
-2. Build it (`gulp run build`)
-3. Run the [Hammerhead playground](https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/playground/server.js) (`npm run http-playground`) to see our proxy in action.
+1. Clone the Hammerhead repository
+    ```cmd
+    git clone https://github.com/DevExpress/testcafe-hammerhead.git
+    ```
+1. Follow to the `testcafe-hammerhead` folder
```

Comment:
Most probably, `Follow` is good and correct here, but I'm not 100% sure. So let's use a more neutral `Go`.

```
1. Go to the `testcafe-hammerhead` folder
```